v20.2/architecture/replication-layer.md

```diff
@@ -79,6 +79,21 @@ A table's meta and system ranges (detailed in the [distribution layer](distribut
 
 However, unlike table data, system ranges cannot use epoch-based leases because that would create a circular dependency: system ranges are already being used to implement epoch-based leases for table data. Therefore, system ranges use expiration-based leases instead. Expiration-based leases expire at a particular timestamp (typically after a few seconds). However, as long as a node continues proposing Raft commands, it continues to extend the expiration of its leases. If it doesn't, the next node containing a replica of the range that tries to read from or write to the range will become the leaseholder.
 
+#### How leases are transferred from a dead node
+
+When a node disconnects, the process by which each of its leases is transferred to a healthy node is as follows:
+
+1. The cluster waits 9 seconds for the dead node's liveness record, which is stored in a system range, to expire.
```

Comment:
I would say a liveness record has an expiration time of 9s since last hearbeat, so the time you wait until expiration after a disconnect varies and on average would be 4.5s.

Reply:
Revised to "The dead node's liveness record, which is stored in a system range, has an expiration time of 9 seconds.  When the node dies, the amount of time the cluster has to wait for the record to expire varies, but on average is 4.5 seconds."